assets/app/scripts/controllers/projects.js

```diff
@@ -19,9 +19,11 @@ angular.module('openshiftConsole')
     });
     AlertMessageService.clearAlerts();
 
-    $('#openshift-logo').on('click.projectsPage', function() {
-      // Force a reload. Angular doesn't reload the view when the URL doesn't change.
-      $route.reload();
+    $timeout(function() {
```

Comment:
I agree, but the listener only applies to this controller even though the ID is now part of a directive used elsewhere. The click listener is a bit of a hack to begin with.